book/src/recursive.md

```diff
@@ -94,4 +94,4 @@ The [SLG solver] in contrast starts by exploring `?X = u32` and finds
 that it works, and then later tries to explore `?X = i32` and finds that it
 fails (because `i32: B` is not true).
 
-[SLG solver]: ./engine.md
\ No newline at end of file
+[SLG solver]: ./engine.md
```

Comment:
???

Reply:
Probably a trailing newline at the end of the file?

Reply:
jup, my PR added a trailing newline :thinking: I expected `github` do add a :x: symbol
in the diff here.

Most other files already have a trailing newline, so I will keep this PR as is